scripts/build/build.js

```diff
@@ -61,6 +63,12 @@ shell.echo();
 
 // --- Docs ---
 
+shell.echo("Updating readme toc...");
+const readmeFilename = path.resolve(__dirname, "../../README.md");
+const oldReadme = fs.readFileSync(readmeFilename, "utf8");
+const newReadme = toc.insert(oldReadme, { maxdepth: 3, bullets: ["*", "+"] });
+fs.writeFileSync(readmeFilename, newReadme);
```

Comment:
I was going to include a 4th level headers in the TOC so the API and CLI option headings could be jumped to.  However, the TOC was quite large at that point.

I figured might as well keep the auto generated TOC though.  The generated TOC matches the current TOC exactly.

Reply:
That's awesome that it can be automatically be generated.

Do you mind creating a different command for it. The build here is a really bad name, this is the script that creates a new release. We update the readme off the release cycle most of the time.

Reply:
Done